src/VisualStudio/CSharp/Impl/Microsoft.VisualStudio.LanguageServices.CSharp.csproj

```diff
@@ -60,14 +60,44 @@
       <DesignTime>True</DesignTime>
       <DependentUpon>CSharpVSResources.resx</DependentUpon>
     </Compile>
+    <Compile Update="Options\AdvancedOptionPage.cs">
+      <SubType>Code</SubType>
+    </Compile>
```

Comment:
📝 These should not be added in this commit

Reply:
i keep reverting these, and VS keeps changing them back...  will manually revert outside of VS.  but i'm not sure what's up here.

Reply:
Still needs to be reverted.